app/presenters/courses_presenter.rb

```diff
@@ -60,6 +60,10 @@ def word_count
     WordCount.from_characters courses.sum(:character_sum)
   end
 
+  def student_word_count
+    WordCount.from_characters CoursesUsers.where('user_id = ?', current_user).sum(:character_sum_ms)
```

Comment:
For all these new individual methods, they are based on `current_user`. How will that work for viewing another user's profile? Maybe we should instead pass an additional optional argument, `user:`, to represent the user whose courses are being presented. And then these methods might be called `individual_word_count` and so on, perhaps.